hvplot/tests/testgeo.py

```diff
@@ -174,6 +183,45 @@ def test_plot_with_specific_gv_tile_obj(self):
         self.assertIsInstance(plot.get(0), gv.element.WMTS)
 
 
+class TestAnnotationNotGeo:
+
+    @classmethod
+    def setup_class(cls):
+        import hvplot.pandas  # noqa
+
+    def test_plot_tiles_doesnt_set_geo(self, simple_df):
+        plot = simple_df.hvplot.points('x', 'y', tiles=True)
+        assert len(plot) == 2
+        assert isinstance(plot.get(0), hv.Tiles)
+        assert 'openstreetmap' in plot.get(0).data
+        bk_plot = bk_renderer.get_plot(plot)
+        assert bk_plot.projection == 'mercator'
```

Comment:
This is true if the test is executed on its own. When the whole module runs, GeoViews is imported and this comes with some side-effects, `bk_plot.projection` becomes Cartopy CRS (Mercator). I'm not sure this side-effect is desired.